apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerVideoViewModel.swift

```diff
@@ -6,22 +6,20 @@ import TwilioVideo
 
 /// Speaker abstraction so the UI can handle local and remote participants the same way.
 struct SpeakerVideoViewModel {
-    let identity: String
-    let displayName: String
-    let isYou: Bool
-    let isMuted: Bool
-    let isDominantSpeaker: Bool
-    let dominantSpeakerStartTime: Date
+    var identity = ""
+    var displayName = ""
+    var isYou = false
+    var isMuted = false
+    var dominantSpeakerStartTime: Date = .distantPast
+    var isDominantSpeaker = false
     var cameraTrack: VideoTrack?
-    var shouldMirrorCameraVideo: Bool
+    var shouldMirrorCameraVideo = false
```

Comment:
With SwiftUI I'm finding it often makes sense to use vars because:

1. It must be a var to bind the value to a view.
2. In a situation like `PresentationLayoutViewModel` where I need "empty" values it is convenient to use default values as done here.

At times SwiftUI can feel a little less type safe and also like there is a little duplicate state in the model to expose everything properly to the UI. But a small price to pay for the benefits of SwiftUI.